utils/fieldToConceptIdMapping.js

```diff
@@ -167,4 +167,66 @@ module.exports = {
      collectionDateTimeStamp: 678166505,
      collectionCardFlag: 137401245,
      collectionAddtnlNotes: 260133861,
-};
\ No newline at end of file
+
+     // participant deceased data
+    participantDeceased: 857217152,
+    participantDeceasedNORC: 987563196,
+    participantDeceasedTimestamp: 772354119,
+
+    // participant profile
+    firstName: 399159511,
+    preferredName: 153211406,
+    middleName: 231676651,
+    lastName: 996038075,
+    suffix: 506826178,
+    homePhone: 438643922,
+    cellPhone: 388711124,
+    otherPhone: 793072415,
+    prefEmail: 869588347,
+    additionalEmail1: 849786503,
+    additionalEmail2: 635101039,
+    address1: 521824358,
+    address2: 442166669,
+    city: 703385619,    
+    state: 634434746,
+    zip: 892050548,
+    canWeVoicemailMobile: 271757434,
+    canWeVoicemailHome: 187894482,
+    canWeVoicemailOther: 983278853,
+    canWeText: 646873644,
+    userProfileHistory: 569151507,
+    userProfileHistoryTimestamp: 371303487,
+    profileChangeRequestedBy: 611005658,
+
+    // Primary cancer site IDs. participant profile -> occurrences {object} (637153953) -> primary site of cancer (740819233). Null until participant diagnosed with cancer.
+    cancerOccurrence: 637153953,
+    primaryCancerSite: 740819233,
+    cancerSitesMap: {
```

Comment:
Should we call it `cancerSites`? `Map` is a type of special object.

Reply:
Sounds good, updated. There are no other refs to this yet - it'll be part of the upcoming RCA variables update just in case you're looking for its use.

Reply:
Sounds good, updated.

There are no other refs to this yet - it'll be part of the upcoming RCA variables update, just in case you're looking for its use.